pkg/kubectl/service.go

```diff
@@ -57,7 +57,7 @@ func paramNames() []GeneratorParam {
 		// port will be used if a user specifies --port OR the exposed object
 		// has one port
 		{"port", false},
-		// ports will be used iff a user doesn't specify --port AND the
+		// ports will be used if a user doesn't specify --port AND the
```

Comment:
@supereagle @MarkusTeufelberger  Thank you, I will use https://en.wikipedia.org/ as my dict.
It feels good to learn new words in community. =)